### ISSUE TITLE
[Clinic] Lab test requests is available without requiring checkin

### ISSUE THREAD

**Issue by valvijo** [esaude/esaude-emr-poc#492]
Actual Results
--
The system is making `Requisição de Laboratório` available without requiring checkin

Expected results
--
The tab `Requisição de Laboratório` in clinic module should only be available only after checkin.

Steps to reproduce
--
Login > Clinic module > Search and select a not checked in patient > Requisição de Laboratório

Screenshot/Attachment (Optional)
--
A visual description of the unexpected behaviour.
![image](https://user-images.githubusercontent.com/3810265/39170140-b86741dc-479a-11e8-8e5d-edd5723ee1d0.png)

**Comment by edrisse** [comment]
@valvijo it is available but no test orders can be created. Save button is disabled when the patient is not checked in. Do we really need to disable the entire screen?

**Comment by valvijo** [comment]
I see two usability issues with the current approach:
(1) The user will only realize that he cannot save the data after waisting his precious time filling the form,
(2) The user will not now why the data cannot be saved. Will take that as a bug

We don't have to hide the entire form but something like:
I see two usability issues with the current approach:
![image](https://user-images.githubusercontent.com/3810265/39171033-dff9bbc8-479d-11e8-93ed-4d2b25ad8a10.png)
